frontend/src/components/Navbar/MeDropdown.tsx

```diff
@@ -110,6 +116,10 @@ function MeDropdown({ isExpanded, setIsExpanded }: Props) {
             >
               Feedback
             </DropdownItem>
+            {/* Release notes */}
+            <DropdownItem icon={FcNews} to="/releases">
+              Release Notes
+            </DropdownItem>
```

Comment:
I think it's better to not put it in the navbar, but we should add a link in the "About" page header

Reply:
https://release-pages.preview.coursetable.com/about next to "Join Us" and "About"